src/main/kotlin/mate/academy/model/impl/Circle.kt

```diff
@@ -0,0 +1,14 @@
+package mate.academy.model.impl
+
+import mate.academy.model.Color
+import mate.academy.model.AreaCalculator
+import mate.academy.model.Figure
+
+class Circle(color: Color, private val radius: Double) : Figure(color), AreaCalculator {
```

Comment:
It is better to Figure() implements AreaCalculator interface instead pass it to each figure-type class